elixir/apps/web/lib/web/components/core_components.ex

```diff
@@ -52,9 +52,9 @@ defmodule Web.CoreComponents do
     The lazy brown fox jumped over the quick dog.
   </.code_block>
   """
-  attr :id, :string, required: true
```

Comment:
Please rollback those changes, your editor did not pick up project formatting rules.

Reply:
I made the changes, now I believe that's OK.